benchmarks/2019.json

```diff
@@ -4471,10 +4525,10 @@
     "isToppedOut": true,
     "deciles": [
       0,
-      78.72,
-      91.67,
-      95.65,
-      97.38,
+      69.86,
```

Comment:
Pretty dramatic change

Reply:
yes but in the favor of the submitter. it lowers the total score to get more than 3 points.